reactioncommerce/reaction
# [Marketplace] Inviting someone to create a new shop or affiliate shop should automatically grant that user appropriate permissions when registered

Resolves #2186 

How to Test:

* Pull and run this branch. Log in as Marketplace Owner.
* Setup your email config (if you don't have it)
* Go to Shop Settings in the dashboard. 
* Open the Invite Owner Form under the marketplace plugin section
* Fill the form (Use a valid email so you can follow the link that gets sent to register)
* Follow link in email and on completing the "reset password" form, check that the new account have access to dashboard actions for the new shop.
